docs/scsb/dump_files.md

```diff
@@ -0,0 +1,13 @@
+### Fetch and process the SCSB files into dump files
+
+SSH to a bibdata machine as deploy user (Find a worker machine in your [environment](https://github.com/pulibrary/bibdata/tree/main/config/deploy)).
+```
+$ tmux attach-session -t full-index
+$ cd /opt/bibdata/current
+$ bundle exec rake scsb:import:full
+CTRL+b d (to detach from tmux)
+```
+This kicks off an import job which will return immediately.  This can be monitored in [sidekiq busy queue](https://bibdata.princeton.edu/sidekiq/busy) or [sidekiq waiting queue](https://bibdata.princeton.edu/sidekiq/queues/default)
+
+Takes 24-25 hours to complete. As they download and unpack they will be placed
```

Comment:
The time it takes to complete has changed. we need to monitor it. The job I ran last week took 12 hours to complete. The previous job that ran through cron it took 22 hours.

Reply:
The job from yesterday took 11 hours. I would change line 12 to say `Takes 11-12 hours to complete.`